src/mars.d

```diff
@@ -1,10 +1,15 @@
-// Compiler implementation of the D programming language
-// Copyright (c) 1999-2015 by Digital Mars
-// All Rights Reserved
-// written by Walter Bright
-// http://www.digitalmars.com
-// Distributed under the Boost Software License, Version 1.0.
-// http://www.boost.org/LICENSE_1_0.txt
+/**
```

Comment:
IMO it's a bit superfluous in the docs, as someone coming accross this code will most likely do it on Github or the dlang website. Even if the person didn't knew the origin of the file, it would be trivial to google it.
Nevertheless, I added it, and we can revisit if there is a need when more modules documentation gets added.